### ISSUE TITLE
Latest Release Recognized as Threat by Windows

### ISSUE THREAD

**Issue by patrickmelix** [Jonathan-LeRoux/IguanaTex#60]
The latest 1.61 release is recognized by Windows as _Trojan:Script/Phonzy.B!ml_ and installation is blocked.

**Comment by Jonathan-LeRoux** [comment]
Is this Windows Defender? Is there a way to force the installation and/or whitelist the file?
We've had many false positives in the past and some success getting previous versions whitelisted by various antivirus software.
You can try downloading the .pptm and creating your own .ppam from it, it might work.

**Comment by patrickmelix** [comment]
Yes, it is Windows Defender. One can force the file to be restored in the settings of the Defender. I mainly wanted to let you know that this is a thing. The previous version is not detected as malicious. I have not tried to create the ppam myself, as I don't currently have the need for the latest version. But I might try later.
As I don't have any experience with this kind of problems I sadly cannot help you with getting this whitelisted... But thanks a lot for your great work!

**Comment by Jonathan-LeRoux** [comment]
Thanks a lot for letting me know! I use Defender as well so I can see if it gets flagged on my end too. I will also check virustotal.

**Comment by Jonathan-LeRoux** [comment]
I just tried downloading the `.ppam` file from the release and double-clicking it. I had to first "unblock" it in its Properties in Explorer, then enable macros, but I didn't get any warning from Windows Defender. I did to scan the file, and Defender didn't report any threat. I don't know if it's because my virus definitions are older or newer than yours...
Note that virustotal.com only reports [one security vendor as detecting a malware](https://www.virustotal.com/gui/file/276e3426a5312fa53874989c4e9b54bd1e6abdbb6c1d0a0a5bec64d381dcf121/detection).
My guess is that these detections are very fuzzy, and it's just a fluke.

**Comment by Jonathan-LeRoux** [comment]
The count has now climbed to 13/62. I have honestly no idea what is triggering these detections, other than stupid AI and herd behavior. The "Code Insight" is actually pretty spot on and explains that all the functions that are used are there for a good reason, not a malicious one. The other AI-generated code analysis basically talks about generic things that, indeed, a malicious add-in would do, but also that any add-in that needs to execute external programs and store information in the registry would do too...

**Comment by lobpcg** [comment]
Confirmed the issue. Detected as https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?Name=Trojan%3AScript%2FWacatac.B!ml and blocked. The unblocking requires allowing Wacatac.B!ml. 

I have just submitted the iguanatex_v1_61.ppam as User Opinion: Incorrect detection at https://www.microsoft.com/en-us/wdsi/filesubmission/

**Comment by Jonathan-LeRoux** [comment]
Thanks @lobpcg !
I hope they can clear it. If they do found malware in there, I'd love to hear how it got in :D

**Comment by jamiescottie1** [comment]
Can confirm that the issue still persists with Win Defender definitions from 18th April 2024. I also submitted the file to https://www.microsoft.com/en-us/wdsi/filesubmission/ as incorrect detection, hopefully they can clear it soon.

I can also confirm that downloading the .pptm, then "Save as" to .ppam seems to work, Defender does not detect the created .ppam as threat in that case.

**Comment by Jonathan-LeRoux** [comment]
Thanks for the updates! I will mention all this in the release.

**Comment by fawidmer** [comment]
Confirming detection of [Wacatac.B!ml](https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?Name=Trojan%3AScript%2FWacatac.B!ml) on Windows 11 (10.0.22631).
Workaround with downloading pptm and save as ppam is working for now. :)

**Comment by hmakmur** [comment]
The same issues also shows with Cisco AMP where it quitely delete files and not tell users.
![ppam](https://github.com/Jonathan-LeRoux/IguanaTex/assets/32586402/920dccb4-13b6-4457-8278-97ef4297f3d7)

**Comment by Jonathan-LeRoux** [comment]
Can you report false positives to Cisco?
Also, what happens if you download the .pptm and save it as .ppam?

**Comment by hmakmur** [comment]
I did not try to play the rename trick but I suspect the result is the same. I can't really report to Cisco. Cisco recommend whitelisting the files. Here is what is written about it, incase you want to see [full details of this file. ](https://www.virustotal.com/gui/file/276e3426a5312fa53874989c4e9b54bd1e6abdbb6c1d0a0a5bec64d381dcf121/detection/f-276e3426a5312fa53874989c4e9b54bd1e6abdbb6c1d0a0a5bec64d381dcf121-1717153293)

**Comment by Jonathan-LeRoux** [comment]
If you get to trying to download the .pptm and converting it to a .ppam via PowerPoint's "save as" ("export" on Mac), I'd be curious to know if that worked.
I'm aware of virustotal's report. Their AI-based "code insight" is pretty spot on, too bad some other vendors flag it as a threat.

**Comment by hmakmur** [comment]
I downloaded the .pptm file, Exported to .ppam file. Scan it with Cisco AMP. The result is fine. No detection.

**Comment by Jonathan-LeRoux** [comment]
Thanks for confirming. I updated the release text to encourage more users to try this.

**Comment by stet-stet** [comment]
Hi. I'd like to report that the pptm to ppam workaround no longer works, at least on 365. After opening the .pptm file I am blocked from editing it; powerpoint redirects me to this [microsoft page]( https://support.microsoft.com/en-us/topic/a-potentially-dangerous-macro-has-been-blocked-0952faa0-37e7-4316-b61d-5b5ed6024216 )

**Comment by Jonathan-LeRoux** [comment]
Can you unblock it as explained on the page?
